sequencer/src/api.rs

```diff
@@ -348,10 +348,20 @@ pub mod test_helpers {
             builder_port: Option<u16>,
         ) -> Self {
             let mut cfg = TestConfig::default_with_l1(l1);
+            cfg.builder_port = builder_port;
 
-            let (builder_task, builder_url) = run_test_builder(builder_port).await;
+            Self::with_state_config(opt, state, persistence, catchup, cfg).await
+        }
 
-            cfg.set_builder_url(builder_url);
+        pub async fn with_state_config(
+            opt: Options,
+            state: [ValidatedState; TestConfig::NUM_NODES],
+            persistence: [impl PersistenceOptions<Persistence = P>; TestConfig::NUM_NODES],
+            catchup: [impl StateCatchup + 'static; TestConfig::NUM_NODES],
+            mut cfg: TestConfig,
```

Comment:
Same as above. `cfg` doesn't help us understand what this is or how it is being used. It is difficult to follow through the lengthy function. It is also inconsistent w/ the the other constructors where it is called `config` (which isn't much better). Maybe call it  `test_config`. (Honestly, `test_config` isn't great either,  but it would be a small improvement). Whatever it's name it should be consistent w/ the other constructors.

Reply:
https://github.com/EspressoSystems/espresso-sequencer/pull/1583/commits/f9dbdcd2ce9c7107659f779cc1520368010352d3
Renamed `cfg` to `config`.
I think either `cfg` or `config` is clear enough.